packages/transport/src/core/CoreErrors.ts

```diff
@@ -14,6 +14,20 @@ class Relationships {
         );
     }
 
+    public alreadyExists(status: RelationshipStatus) {
```

Comment:
I'd rename the error to `relationshipAlreadyExists` for clarity.

Reply:
This error will be thrown as `error.transport.relationships.alreadyExists`, as `alreadyExists` is just the name of the error's suffix. Do you still want to rename it, so that `error.transport.relationships.relationshipAlreadyExists` will be thrown?